easybuild/easyconfigs/t/TRIQS/TRIQS-2.1.1-foss-2018b-Python-2.7.15.eb

```diff
@@ -0,0 +1,52 @@
+easyblock = 'CMakeMake'
+
+name = 'TRIQS'
+version = '2.1.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://triqs.github.io/'
+description = """
+ TRIQS (Toolbox for Research on Interacting Quantum Systems) is a
+ scientific project providing a set of C++ and Python libraries to
+ develop new tools for the study of interacting quantum systems.
+"""
+
+docurls = ['https://triqs.github.io/triqs/2.1.x/reference.html']
+software_license = 'LicenseGPLv3'
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+toolchainopts = {'pic': True, 'usempi': True}
+
+sources = ['%(version)s.tar.gz']
+source_urls = ['https://github.com/TRIQS/triqs/archive/']
+checksums = ['f76909c02bf306d0796297d9854ad5f1262f2cbd6b8c8521760e972dd72f9d13']
+
+dependencies = [
+    ('Python', '2.7.15'),
+    ('Boost', '1.67.0'),
+    ('Clang', '7.0.1'),
+    ('GMP', '6.1.2'),
+    ('Mako', '1.0.7', versionsuffix),
+    ('h5py', '2.8.0', versionsuffix)
+]
+
+builddependencies = [
+    ('CMake', '3.11.4')
+]
+
+separate_build_dir = True
+
+runtest = 'test'
+
+sanity_check_paths = {
+    'files': ['lib/libtriqs.%s' % SHLIB_EXT],
+    'dirs': ['bin', 'include', 'lib', 'share'],
+}
```

Comment:
add a sanity check command to ensure that the Python component is installed correctly?

```python
sanity_check_commands = ["python -c 'import pytriqs'"]